### PR TITLE
Travel time exponential

### PR DESCRIPTION
**Pull request by DanielHettegger**

### CODE REVIEW

File: imp_act/environments/print_episode.py
Comment (moratodpg) [inline imp_act/environments/print_episode.py:1]: Nice addition!

**Comment by DanielHettegger** [comment]
The final formula I decided to implement is: 
$c_t = p ((\frac{t_t}{t_b})^q -1) $ (clipped at 0)
This has the nice property of starting at 0 cost for no delays while still exhibiting exponential behavior with a gradient of at least 1.

**Review by moratodpg — COMMENTED** [review]
I checked it and enjoyed the implementation.
Pytests also passing!

However, the significance of the delays is still minor? (Or maybe we have to tune the factors first?)
See below what I got when I printed out the results.
image

**Comment by DanielHettegger** [comment]
> However, the significance of the delays is still minor?

Yes, I left the factor at 1, so we can tune it once we have the rest of the environment updated. It should probably be in the millions later.

**Comment by moratodpg** [comment]
Then, all is working well! :) We can just tune it later.

**Comment by moratodpg** [comment]
I tuned the factor to -500_000 and the value is still very small:
image

File: imp_act/environments/print_episode.py [hunk 1]
```diff
@@ -3,23 +3,60 @@
...
+
+def heuristic_policy(observation):
+    actions = []
+    current_time = observation["time_step"]
+    for edge in observation["edge_observations"]:
+        edge_actions = []
+        for segment in edge:
+            if segment >= 5:
+                edge_actions.append(4)  # Reconstruction
+            elif segment >= 5:
```
Comment (omniscientoctopus) [inline imp_act/environments/print_episode.py:29]: Minor comment: Since we only have 5 states {0,1,2,3,4} `segment >= 5` can be removed?

Comment (DanielHettegger) [reply imp_act/environments/print_episode.py:29]: Well, this is just the humble heuristic 😄 
I would leave it in, and we can change the values to test different policies.

**Comment by DanielHettegger** [comment]
> I tuned the factor to -500_000 and the value is still very small: image

I think given that a single replacement is 35 million and delays are in the single-digit percentages, we will need to set the tuning factor quite a bit higher than that 😄 

Maybe something relative to the budget we give.

**Pull request closed without merging**